scripts/qa_crowbarsetup.sh

```diff
@@ -4586,6 +4586,14 @@ function onadmin_prepare_crowbar_upgrade()
         # move nodes to upgrade mode
         safely curl -s -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/prepare.json
     else
+        if crowbarctl upgrade prechecks --format plain | grep "false" ; then
+            complain 11 "Some necessary check before the upgrade has failed"
+        fi
+
+        # FIXME crowbatctl is currently missing admin repo check
```

Comment:
we just released a new rubygem for `crowbar-client`, so the subcommand should be in the product today.